code/modules/fallout/machinery/doors/unpowered.dm

```diff
@@ -318,3 +318,141 @@
 
 /obj/machinery/door/unpowered/securedoor/outlaw/boss
 	req_access_txt = "142"
+
+
+// ------------------------------------
+// Badlands
+// GMB
```

Comment:
You forgor the lower line

Reply:
Standard for the rest. Can change 'em if needed.